audit/projects/kubernetes-public/secrets/k8s-infra-build-clusters-kubeconfig/versions.json

```diff
@@ -1,11 +1,20 @@
 [
+  {
+    "createTime": "2021-09-24T15:57:10.612313Z",
+    "etag": "\"15ccbfccd0d959\"",
+    "name": "projects/127754664067/secrets/k8s-infra-build-clusters-kubeconfig/versions/2",
+    "replicationStatus": {
+      "automatic": {}
+    },
+    "state": "ENABLED"
+  },
```

Comment:
Manually updated with a new version of the secret to fix : https://github.com/kubernetes/k8s.io/issues/2559